Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
+#include <QTime>
```

Comment:
Do you need this? Your .cpp already has this include.

If you do, you should forward declare it instead.
https://stackoverflow.com/questions/4757565/what-are-forward-declarations-in-c